stable/moodle/templates/deployment.yaml

```diff
@@ -18,6 +18,15 @@ spec:
         app: {{ template "moodle.fullname" . }}
         chart: {{ template "moodle.chart" . }}
         release: "{{ .Release.Name }}"
+{{- if or .Values.podAnnotations .Values.metrics.enabled }}
```

Comment:
`podAnnotation` is not defined in `values.yaml` (and `values-production.yaml` if it exists).

Can you add it and also include the new variable in the README?